lib/vmpooler.rb

```diff
@@ -79,6 +87,14 @@ def self.new_graphite(server)
     end
   end
 
+  def self.new_statsd(server, port)
+    if server.nil? or server.empty? or server.length == 0
```

Comment:
I'll fix this one up, although we do expect the value to default to nil if the configuration file is invalid. https://github.com/shermdog/vmpooler/blob/8d75865a5ce034c41d41612d0d3c1dec80f1118f/vmpooler#L14